backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -24,6 +25,13 @@ def _owner_or_allowed(user):
     return None if has_scope("write:collections") else user
 
 
+def _get_doi_from_body(body):
+    links = body.get("links", [])
+    doi_list = [link["link_url"] for link in links if link["link_type"] == "DOI"]
+    # TODO (ebezzi): remove DOI normalization when all DOIs have been migrated to CURIE
+    return Collection._normalize_doi(doi_list[0]) if doi_list else None
```

Comment:
out-of-scope for PR, but if normalization means generating a CURIE, maybe rename method to `_to_curie` (`normalize` means I have to read the method)

Reply:
Yes, this code will change as soon as we decide how to normalize DOIs. That requires frontend changes though, so it will be done at a later date.